### DISCUSSION TITLE
i want to know how to add 'delay' attribute to component,i want a example,thanks.

### DISCUSSION THREAD

**Discussion by danzhoua** [Akryum/v-tooltip#328] [Q&A] [answered]

**Comment by jwmoreland** [comment, marked as answer]
Here's an example how I did it with a content and a delay written in. 
`<button v-tooltip="{ content: 'You have ' + count + ' new messages.', delay: { show: 500, hide: 100} }">`
**Updated** per TomDeSmet's suggestion

**Comment by TomDeSmet** [comment]
I think there should be a `,` before delay
